src/index.html

```diff
@@ -16,21 +16,28 @@
                 <li class="Subcribe">Subscribe</li>
             </ul>
         </nav>
+        <nav class="counter">
+            <h3> Movies (<span id="movie-counter"></span>)</h3>
+            <h3>Favourite</h3>
+          </nav>
     </header>
 
     <section class="main">
+        <div class="alert"></div>
+
+
         <div class="main-wrapper">
 
         </div>
     </section>
     <footer>
         <div class="designers">
           <p class="name"></p>
```

Comment:
- [ ] Great implementation of your footer. However, kindly check this line as it does not perform any function to the code.

It is important to rid your code of unused lines to ensure Clarity and Readability of your code and also ensure the best performance of your application. 🧹